src/ontime/model/libs/skforecast/forecaster_autoreg_multi_variate.py

```diff
@@ -0,0 +1,29 @@
+from abc import ABCMeta
+
+from ontime.abstract.abstract_base_model import AbstractBaseModel
+from ontime.time_series import TimeSeries
+
+from skforecast.ForecasterAutoregMultiVariate import (
+    ForecasterAutoregMultiVariate as SkForecastForecasterAutoregMultiVariate,
+)
+
+
+class ForecasterAutoregMultiVariate(AbstractBaseModel):
+    """
+    Generic wrapper around SkForecast ForecasterAutoreg models
+    """
+
+    def __init__(self, model, **params):
+        super().__init__()
+        if isinstance(model, ABCMeta):
+            model = model()
+        self.model = SkForecastForecasterAutoregMultiVariate(regressor=model, **params)
```

Comment:
ForecasterAutoregMultiVariate takes other series into account instead of ForecasterAutoregMultiSeries BUT it allows only one series to be predicted and must know how many steps are going to be predicted